.github/workflows/build.yml

```diff
@@ -126,24 +126,24 @@ jobs:
       id: integration_tests
       run: PRODUCT_SPRING_BOOT_VERSION=${{ matrix.spring-boot-version }} ./mvnw -B -f sdk-tests/pom.xml verify
     - name: Upload test report for sdk
-      uses: actions/upload-artifact@master
```

Comment:
Why change this from `master` to `v3.0.0` - did you encounter an issue that warranted this change?